crates/sui-json-rpc/src/read_api.rs

```diff
@@ -746,6 +746,41 @@ impl ReadApiServer for ReadApi {
     async fn get_checkpoint(&self, id: CheckpointId) -> RpcResult<Checkpoint> {
         Ok(self.get_checkpoint_internal(id)?)
     }
+
+    async fn get_checkpoints(
+        &self,
+        cursor: Option<CheckpointSequenceNumber>,
+        limit: Option<usize>,
+        descending_order: bool,
+    ) -> RpcResult<CheckpointPage> {
+        let limit = cap_page_limit(limit);
```

Comment:
I think this requires a much lower limit(e.g., 20), can we define a different constant?